### ISSUE TITLE
Remove instance.validate in Monky.prototype.build

### ISSUE THREAD

**Issue by bakorer** [behrendtio/monky#32]
I want to remove `instance.validate` in `Monky.prototype.build` method.

- https://github.com/behrendtio/monky/blob/0170867/lib/monky.js#L124-L126

Because the `instance.validate` method is not called when the mongoose instance is created (not saved), right?

For example, I can't use `monky` if I write below test codes.

```js
/*
 * Model file
 */
var mongoose = require('mongoose');
var UserSchema = new mongoose.Schema({ username: { type: 'string' }});

UserSchema.path('username').validate(function(value) {
  return validation.required(this, value);
}, 'required');

UserSchema.virtual('current_username').get(function() {
  if (this.username) {
    return this.username;
  } else {
    return 'temp username';
  }
});

mongoose.model('User', UserSchema);


/*
 * Pre-test file (setup)
 */
monky.factory('User', { username: 'foo' });


/*
 * Actual test
 */
describe('User', function() {
  describe('current_username virtual', function() {

    it('should return the string `temp username` if username is undefined', function(done) {
      monky.build('User', { username: null }, function(err, user) {
        console.log(err);
        /**
        { message: 'Validation failed',
          name: 'ValidationError',
          errors: 
           { reply_to: 
              { message: 'required',
                name: 'ValidatorError',
                path: 'username',
                type: 'user defined',
                value: null } } }
        **/

        console.log(user);
        // undefined

        user.current_username.should.equal 'temp username'
      });
    });
  });
});
```

In this case, I have to use `new User()` instead of `monky.build("User", function(err, user){ ...` :cry: 
I think it bad code because I cannot use the define of monky and there are some defines to create `User` instance.

**Comment by behrendtio** [comment]
Alright, it's in `master`. Would you mind testing this branch first, before I release it? Thanks.

**Comment by bakorer** [comment]
@behrendtio Thx :exclamation:  I tested with master branch. It works :+1:

**Comment by behrendtio** [comment]
@bakorer Nice, thanks. I just released `0.6.7` which includes these changes.

**Comment by bakorer** [comment]
@behrendtio Thanks a lot :smile: I will use `0.6.7` :exclamation:
